Helper/OneLogHelper.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types=1);
+
+namespace KoderHut\OnelogBundle\Helper;
+
+use KoderHut\OnelogBundle\OneLog;
+
+/**
+ * Class OneLogHelper
+ *
+ * @author Joao Jacome <969041+joaojacome@users.noreply.github.com>
+ */
+class OneLogHelper
```

Comment:
I would also rename this to OneLogStatic instead of OneLogHelper. seems more appropriate since we are using this we would use the instance even though this is a proxy to the instance itself